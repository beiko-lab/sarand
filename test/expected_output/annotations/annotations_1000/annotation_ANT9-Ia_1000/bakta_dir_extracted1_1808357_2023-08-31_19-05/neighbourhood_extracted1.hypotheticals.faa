>KIGIOP_00005 hypothetical protein
MPQHIVQKFLGHESPEMTSRYAHIFDETLKNEFTKFQEKLVTNNGDVLDLDEDNEVDDVELQWFKKNINAQVLPNGYCRLPVVAGGCPHANACLDCTHFCTSKQFLPQHEEQLERTEELLAIAKDKQWQRQVETNSRVKERLEQIIGSLTG
>KIGIOP_00010 hypothetical protein
MDKQVRNTTEIVRLAKQKSKKTREKVDKAISKFSIEGKVINFNSIAKEANVSKSWLYKEHDIRQRIESLRERQITANVVSKPKKSSRSEEILIKTLKRRVMELEKENKKLQNQIQKLYGDLYNKE
