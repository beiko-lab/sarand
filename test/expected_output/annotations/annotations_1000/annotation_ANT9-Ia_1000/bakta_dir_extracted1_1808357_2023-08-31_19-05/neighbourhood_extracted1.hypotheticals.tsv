#Annotated with Bakta v1.8.1, https://github.com/oschwengers/bakta
#Database v5.0, https://doi.org/10.5281/zenodo.4247252
#Sequence Id	Start	Stop	Strand	Locus Tag	Mol Weight [kDa]	Iso El. Point	Pfam hits	Dbxrefs
contig_1	11	466	+	KIGIOP_00005	17.4	5.1		
contig_1	473	850	+	KIGIOP_00010	14.8	10.1		
